app/account/hooks/useUploadFile.ts

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react'
+import { createClient } from '@supabase/supabase-js'
+import { updateImageMetadataAction } from '../action/update-image-metadata'
+
+export const useUploadFile = () => {
+  const [isLoading, setIsLoading] = useState(false)
+
+  const uploadFile = async (
+    profile_picture_url: string | null,
+    selectedImage: File,
+    userId: string
+  ) => {
+    setIsLoading(true)
+
+    const supabase = createClient(
+      process.env.NEXT_PUBLIC_SUPABASE_URL!,
+      process.env.NEXT_PUBLIC_API_KEY!
+    )
+
+    if (profile_picture_url) {
+      const parts = profile_picture_url.split('/')
+      const fileName = parts.pop()
+
+      await supabase.storage
+        .from('tabi-memo-uploads')
+        .remove([`account/edit/${fileName}`])
+    }
```

Comment:
I created a new policy to enable the removal of a file in the 'account edit' folder in Supabase storage. I believe that each time a user uploads a profile image, the old one should be removed so that there is only one image stored in the storage.

![Screenshot 2024-02-16 at 16 05 13](https://github.com/tabi-memo/frontend/assets/70562492/3a7e4f62-e193-48c2-9b77-b20138ed09a7)

![Screenshot 2024-02-16 at 16 22 51](https://github.com/tabi-memo/frontend/assets/70562492/7c004366-0f82-40bf-b9e8-96aae288752b)